src/lib.rs

```diff
@@ -381,6 +400,7 @@ impl<S: SuffixScheme> FileRotate<S> {
             ContentLimit::Bytes(bytes) => {
                 assert!(bytes > 0);
             }
+            ContentLimit::Time(_) => {}
```

Comment:
Seeing this, perhaps we should put the assertions inside the creation of the ContentLimit object (in a future commit) @Ploppz ?

Also I think the location of this item should be last in the struct since `Time` supercedes `Lines` lexicographically.

Reply:
I like how currently you can construct ContentLimit directly without a function so I would leave it as it is